app/controllers/line_items_controller.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class LineItemsController < ApplicationController
+  # rubocop:disable Metrics/MethodLength
+  def create
+    chosen_product = Book.find(params[:book_id])
+    current_cart = @current_cart
+
+    if current_cart.books.include?(chosen_product)
+      @line_item = current_cart.line_items.find_by(book_id: chosen_product)
+      @line_item.quantity += 1
```

Comment:
change default value to zero for still using such approach

Reply:
+